app/frontend/src/store/modules/apiAccess.js

```diff
@@ -30,7 +30,14 @@ export default {
     ephemeralPasswordRSAKey: state => state.ephemeralPasswordRSAKey,
     generatedClient: state => state.generatedClient,
     generatedPassword: state => state.generatedPassword,
-    step: state => state.step
+    step: state => state.step,
+    tokenEndpoint(state) {
+      let prefix = 'sso';
+      if (['dev', 'test'].includes(state.environment.toLowerCase())) {
+        prefix += `-${state.environment.toLowerCase()}`;
+      }
+      return `https://${prefix}.pathfinder.gov.bc.ca/auth/realms/jbd6rnxw/protocol/openid-connect/token`;
+    }
```

Comment:
Now that I look at this again, I now realize this is just trying to generate the general openid-connect token endpoint. We already have the base auth url defined at `config.keycloak.serverUrl` (accessible via $config.keycloak.serverUrl) which would return for example `https://sso-dev.pathfinder.gov.bc.ca/auth` and change based on environment already. Assuming I didn't botch how to access that variable, it'd look something like this...

```js
tokenEndpoint(state) {
  return `${this._vm.$config.keycloak.serverUrl}/realms/jbd6rnxw/protocol/openid-connect/token`;
}